packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.Props.ts

```diff
@@ -59,4 +59,27 @@ export interface IBaseAutoFillProps extends React.HTMLAttributes<HTMLInputElemen
    */
   enableAutoFillOnKeyPress?: KeyCodes[];
 
+  /**
+   * the default value to be visible
+   */
+  defaultVisibleValue?: string;
+
+  /**
+   * Handler for checking and updating the value if needed
+   *  in componentWillReceiveProps
+   *
+   * @param {IBaseAutoFillProps} defaultVisibleValue - the defaultVisibleValue that got passed
+   *  in to the auto fill's componentWillReceiveProps
+   * @returns {string} - the updated value to set, if needed
+   */
+  updateValueInWillReceiceProps?: () => string;
```

Comment:
it still feels weird that we are extending BaseAutoFill this way, would composition achieve the same in a more react idiomatic way? Looking at the code, doesn't seem to be that straightforward.